packages/cli/src/commands/process/create.ts

```diff
@@ -22,11 +23,11 @@ export default class ProcessCreate extends Command {
   }]
 
   async run(): Promise<IProcess> {
-    const {args, flags} = this.parse(ProcessCreate)
+    const { args, flags } = this.parse(ProcessCreate)
     const definition = JSON.parse(args.DEFINITION) as IProcess
-    
+
     const { account, mnemonic } = await this.getAccount(flags.account)
-    this.spinner.start('Create process')
+    cli.action.start('Create process')
```

Comment:
all action should have a verb in ..ing
```suggestion
    cli.action.start('Creating process')
```